autogen/main/cluster.tf.tmpl

```diff
@@ -675,6 +677,20 @@ resource "google_container_cluster" "primary" {
       }
 
       local_ssd_encryption_mode = lookup(var.node_pools[0], "local_ssd_encryption_mode", null)
+      containerd_config {
```

Comment:
This change is only under the default node pool's node_config, you will also want to review the one around line 1024 and if this method works for Autopilot clusters.